src/schema/userSchema.ts

```diff
@@ -0,0 +1,101 @@
+import { Schema } from 'mongoose'
+
+const validateEmail = (email: string) => {
```

Comment:
can we move this function to genric helpers in utils?

Reply:
Yes, Will do that